doc/api/globals.markdown

```diff
@@ -2,188 +2,250 @@
-<!-- type=global -->
+See the [Timers documentation][] for more information.
 
-* {Object} The global namespace object.
+### `setTimeout(callback, ms)`
+
+The `setTimeout()` method creates a Timer that will call the `callback`
+function after *at least* `ms` milliseconds. The actual delay depends
+on external factors such as the operating system timer granularity and system
+load.
 
-In browsers, the top-level scope is the global scope. That means that in
-browsers if you're in the global scope `var something` will define a global
-variable. In Node.js this is different. The top-level scope is not the global
-scope; `var something` inside an Node.js module will be local to that module.
+The timeout must be in the range of 1-2,147,483,647 (inclusive). If the value is
+outside that range, the value is changed to 1 millisecond. Broadly speaking, a
+timer cannot span more than 24.8 days.
 
-## module
+The method returns an object representing the timer than can be passed later
+to the `clearTimeout()` method to halt further calling of the `callback`
+function.
+
+See the [Timers documentation][] for more information.
+
+## Module-scope Variables
+
+Module-scoped variables are accessible to all modules but are not true
+globals.
+
+### `__dirname`
 
 <!-- type=var -->
 
-* {Object}
+* {String}
 
-A reference to the current module. In particular
-`module.exports` is used for defining what a module exports and makes
-available through `require()`.
+The name of the directory that the currently executing script resides in.
 
-`module` isn't actually a global but rather local to each module.
+Example: running `node example.js` from `/Users/mjr`
 
-See the [module system documentation][] for more information.
+    console.log(__dirname);
+      // Prints: /Users/mjr
 
-## process
+### `__filename`
 
-<!-- type=global -->
+<!-- type=var -->
+
+* {String}
+
+The filename of the code being executed.  This is the resolved absolute path
+of this code file.  For a main program this is not necessarily the same
+filename used in the command line.  The value inside a module is the path
+to that module file.
+
+Example: running `node example.js` from `/Users/mjr`
+
+    console.log(__filename);
+      // Prints: /Users/mjr/example.js
+
+### `exports`
+
+<!-- type=var -->
+
+A reference to the `module.exports`. See the [Module system documentation][]
+for details on when to use `exports` and when to use `module.exports`.
+
+### `module`
+
+<!-- type=var -->
 
 * {Object}
 
-The process object. See the [`process` object][] section.
+A reference to the current module. In particular, `module.exports` is used to
+define what a module exports and makes available through `require()`.
 
-## require()
+See the [Module system documentation][] for more information.
+
+## Global and Module Scope
+
+Certain variables are defined at *both* the global and module scopes. The
+specific operation of these can vary depending on the current scope in
+which JavaScript code is executing.
```

Comment:
It might not be clear to readers why this difference from module-scoped things matters. I'm not sure it even *does* matter to the majority of users either. I think this paragraph might just confuse some users...